File: ruby/Napakalaki/lib/prize.rb
#encoding: utf-8

#Sofía Almeida Bruno
#María Victoria Granados Pozo

module NapakalakiGame
  class Prize
    def initialize(t, l)
      @treasures = t
      @level = l
    end

    def getTreasures
      @treasures
    end

    def getLevels
      @level
    end

    def to_s
      "\n\tTesoros ganados: #{@treasures} \n\tNiveles ganados: #{@level}"
    end

  end
end

File: java/Napakalaki/src/NapakalakiGame/Prize.java
package napakalaki;

/**
 *
 * @author Sofía Almeida Bruno
 * @author María Victoria Granados Pozo
 */
public class Prize {
    private int treasures;
    private int levels;
    
    /**
     * 
     * Constructor
     * @param t número de tesoros ganados
     * @param l número de niveles ganados
     */
    public Prize(int t, int l) {
        treasures = t;
        levels = l;
    }
    
    /**
     * 
     * Consultor de tesoros
     * @return número de tesoros ganados
     */
    public int getTreasures() {
        return treasures;
    }
    
    /**
     * 
     * Consultor de niveles
     * @return número de niveles ganados
     */
    public int getLevels() {
        return levels;
    }
    
    /**
     * 
     * @return String con la información del objeto
     */
    public String toString() {
        return "Treasures = " + Integer.toString(treasures) +
                "\nLevels = " + Integer.toString(levels);
    }
}
